assets/CSS/shopping-cart.css

```diff
@@ -1 +1,80 @@
-@import 'colors.css';
\ No newline at end of file
+@import "colors.css";
+a {
+  text-decoration: none;
+  color: white;
+}
+.btn-apply1 {
+  color: var(--secondary-color);
+  font-weight: 500;
+  background-color: white;
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+.btn-apply1:hover {
+  color: #4498e6;
+  background-color: white;
+  border: solid 2px #4498e6;
+
+}
+
+.btn-apply1::after {
+  color: var(--secondary-color);
+  font-weight: 500;
+  background-color: white;
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+
+
+
+.btn-apply {
+  color: white;
+  background-color: var(--secondary-color);
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+.btn-apply:hover {
+  color: white;
+  background-color: #4498e6;
+  border: solid 2px var(--secondary-color);
+
+}
+
+.btn-apply:active {
+  color: white;
+  background-color: var(--secondary-color);
+  border: solid 2px var(--secondary-color);
+}
+
+.btn-apply:focus{
+  color: white;
+  background-color: var(--secondary-color);
+  border: solid 2px var(--secondary-color);
+}
+
+
+
+
+
+* {
+  font-family: Poppins;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+
+}
+
+
+.coupon {
+  margin-right: 30px;
+  background-color: var(--bg-color);
+}
+
+
+
+.form-control:focus {
```

Comment:
Please do not change property in the bootstrap  class directly you already have a class coupon